freegeek-info.sh

```diff
@@ -31,7 +34,8 @@ while IFS= read -r gpu; do
             vram=$(lspci -v -s $(lspci | grep -i 'arc' | awk '{print $1}') | grep -i 'prealloc size' | awk '{print $3}' | tr -d 'M')
             if [[ "$vram" =~ ^[0-9]+$ ]]; then
                 vram_gb=$(echo "scale=2; $vram / 1024" | bc)
```

Comment:
```suggestion
                vram_gb=$(echo "scale=2; $vram / 1024" | bc | cut -d'.' -f1)
```
Can't test this rn, but if this works, can skip redefining the variable

Reply:
Same for below...

Reply:
Ill test it.

Yes that works, Ill change it.